W 11673756 manager processboard se (#111)
mulesoft/docs-rpa: rpa-manager/modules/ROOT/pages/processautomation-manage.adoc

|===

* To create or copy a project, you must be xref:usermanagement-manage.adoc#designate-a-user-as-project-manager[designated as project manager].
* To transition a project to another phase, you must have the needed permissions as a member of the <<projectdata-processteam, process team>>.

== Create an Automation project

When a project manager clicks the icon *Start automation* for a process in the *My Backlog* view of the *My RPA* module, RPA Manager creates a project for the automation of the process.

To automate a process already evaluated with a third-party tool, you can create an automation project inside a category directly in the *Process Automation* module:

. Open the *Project Management* view of the *Process Automation* module.
. Click on the name of the category in which you want to create the new project.
. In the *<category name>* view, click *Create*.
. [[form-createprocess]] Complete the form *Create process*:
+
include::rpa-manager::partial$form-automationproject.adoc[]
. Click *Save*.

RPA Manager returns to the *<category name>* view. Your new project is sorted into the view according to your workspace settings.

== Copy an Automation project

> A copy of an automation project contains all information from the original project including the most recent revision of all workflows with the exception of the bot assignments to test and productive configurations.

Reviewer: I am not sure what this means: "to test and productive configurations."

> Copy a project to automate a similar process faster:

Reviewer: ```suggestion
Copy a project to automate a similar more quickly:
```

Author: @dhirsch1 there is a difference between process and project ( see above). Therefor I included the word 'process': '
Copy a project to automate a similar process more quickly:'

. Open the *Project Management* view of the *Process Automation* module.
. Click on the name of the category in which you want to create the new project.
. In the *<category name>* view, click the *Copy* icon image:copy-icon.png[copy symbol,1.5%,1.5%] on the panel of the project you want to copy.
. Check and edit the form *Copy process*:
+
For an explanation of the properties, see <<form-createprocess, *Create process*>>.
. Click *Save*.

> To complete copying of the project, first transfer the project to the *Build* phase. Then open it in the xref:rpa-builder::index.adoc[RPA Builder]. The project then receives its own identification values. The copied project can only transition to the *Test* phase after it has been uploaded from the RPA Builder to the RPA Manager.

Reviewer: Consider using an ordered list here.

To complete copying the project:

1. Transfer the project to the *Build* phase. 
2. Open it in the xref:rpa-builder::index.adoc[RPA Builder]. 
+
The project receives its own identification values.

The *Delete* icon is only displayed if the logged-in user is the project's manager and the process has never been used productively. If the project has been transferred at least once to the production phase and deployed to an RPA Bot for processing, it can no longer be deleted.

To remove a project:

. Open the *Project Management* view of the *Process Automation* module.
. Click on the name of the category of the project.
. In the *<category name>* view, click the *Remove* icon image:delete-icon.png[trash symbol,1.5%,1.5%] on the panel of the project you want to remove.
. Confirm the prompt.

== Manage Process Project Tasks

You can manage all tasks that arise in association with automating a process via an agile process board if you have <<projectdata-useprocessboard, enabled it for the project>>.

> A task consists of a name, a description, an owner, a priority, a due date, and an effort estimate. You can drag tasks across the board to reflect their progress.

Reviewer: ```suggestion
A task consists of a name, a description, owner, priority, due date, and effort estimate. You can drag tasks across the board to reflect their progress.
```

A process board is a table with three columns, across which you can arrange task items:

* *To-do*
+
The *To-do* column contains all the tasks that have not been processed yet. You create new tasks here. If you have <<projectdata-phasereleasetasks, enabled the creation of automatically generated tasks for phase transitions>>, RPA Manager creates a new corresponding task during each phase transition. Remember to move tasks that are being processed into the *In Progress* area.
* *In Progress*
+
The *In Progress* column contains all the tasks that someone is currently working on. Remember to move tasks that have been completed into the *Completed* area.
* *Completed*
+

> The *Completed* column contains all the tasks that have already been completed. You can edit and move completed tasks, too. If you have enabled the creation of automatically generated tasks for the phase transitions, the button for the transition to the next phase disabled until this task has been moved to the *Completed* column.

Reviewer: ```suggestion
The *Completed* column contains all tasks that have already been completed. You can edit and move completed tasks, too. If you have enabled the creation of automatically generated tasks for the phase transitions, the button for the transition to the next phase disabled until this task has been moved to the *Completed* column.
```

. Complete the form *Create new task*:
* *Task Owner*:
+
The task owner is responsible for completing the task. Choose a user from the process team. The task item displays the user's profile picture. RPA Manager shows all tasks of the task owners on their personal *My Process Board* view in the *My RPA* module.
* *Priority*:
+
You can set the priority of the task to low, medium, or high. The task item displays a grey, yellow or red label, respectively.
* *Due date*:
+
Date on which the task should be completed.
* *Effort*:
+
Time the task will approximately take.
. Click Save.

=== Edit a Task

Edit tasks to reflect changed requirements:

. <<openprocessboard, Open a process board>>.
. Click a task item.
. Edit the form *Edit task*.
+
For an explanation of the properties, see <<form-createprocessboardtask, *Create a task*>>.
. Click *Save*

=== Delete a Task

Delete obsolete tasks:

. <<openprocessboard, Open a process board>>.
. Click a task item.
. Click *Delete*.
. Confirm the deletion.

=== Move a Task

Move tasks within a column to sort them or across columns to reflect their progress:

. <<openprocessboard, Open a process board>>.
. Drag a task item to its new place.

> If you have enabled the creation of automatically generated tasks for the phase transitions, move this task to the *Completed* column after all other tasks have been completed to enable the transition of the project to the next phase.

Reviewer: ```suggestion
If you enabled the creation of automatically generated tasks for the phase transitions, move this task to the *Completed* column after all other tasks have been completed to enable the transition of the project to the next phase.
```

== Analyze the Financial Aspects of a Process

Open the xref:processoperations-financeanalysis.adoc[*Finance Analysis* view] to record, monitor, and analyze the financial aspects of a process:

. Open the *Project Management* view of the *Process Automation* module.
. Click on the name of the category of the project.

> . In the *<category name>* view, click the *Finance Analysis* icon image:financeanalysis-icon.png[ symbol,1.5%,1.5%] the panel of the project you want to analyze.

Reviewer: It looks like this is missing an "of" or "for" before "the panel of the project you want to analyze."

Author: ... on ...
